cmd/agent/common/commands/check.go

```diff
@@ -456,6 +461,16 @@ func printMetrics(agg *aggregator.BufferedAggregator) {
 	}
 }
 
+func writeCheckToFile(c check.Check, checkStatus []byte) {
+	_ = os.Mkdir(common.DefaultCheckFlareDirectory, os.ModeDir)
+	flarePath := common.DefaultCheckFlareDirectory + "check_" + string(c.ID()) + "_" + time.Now().Format(time.RFC3339) + ".log"
```

Comment:
Note: It is usually better to use `path.join` (https://golang.org/pkg/path/#Join) instead of assuming the folder name ends with `\` or `/`.